src/editor/Gui.cpp

```diff
@@ -1631,6 +1638,53 @@ void Gui::drawKeyvalueEditor_RawEditTab(Entity* ent) {
 	ImGui::EndChild();
 }
 
+void Gui::drawGOTOWidget() {
+	ImGui::SetNextWindowSize(ImVec2(410, 200), ImGuiCond_FirstUseEver);
+	ImGui::SetNextWindowSizeConstraints(ImVec2(410, 200), ImVec2(410, 200));
+	static vec3 coordinates = vec3();
+	static vec3 views = vec3();
+	if (ImGui::Begin("Go to coordinates:", &showGOTOWidget, 0)) {
+		ImGuiStyle& style = ImGui::GetStyle();
+		float padding = style.WindowPadding.x * 2 + style.FramePadding.x * 2;
+		float inputWidth = (ImGui::GetWindowWidth() - (padding + style.ScrollbarSize)) * 0.33f;
+		if (showGOTOWidget_update)
+		{
+			coordinates = app->cameraOrigin;
+			views = app->cameraAngles;
+			showGOTOWidget_update = false;
+		}
+		ImGui::Text("Coordinates");
+		ImGui::PushItemWidth(inputWidth);
+		ImGui::DragFloat("##xpos", &coordinates.x, 0.1f, 0, 0, "X: %f");
+		ImGui::SameLine();
+		ImGui::DragFloat("##ypos", &coordinates.y, 0.1f, 0, 0, "Y: %f");
+		ImGui::SameLine();
+		ImGui::DragFloat("##zpos", &coordinates.z, 0.1f, 0, 0, "Z: %f");
+		ImGui::PopItemWidth();
+		ImGui::Text("Views");

```

Comment:
rename to "Angles". Mappers/ripenters will understand that better.

Reply:
Ok

Reply:
the button got renamed to "Angles" instead of this "Views" text. "GOTO" makes sense for the button.

Reply:
Yes)) Now fixed ?